packages/block-library/src/heading/edit.native.js

```diff
@@ -24,6 +24,24 @@ import './editor.scss';
 const minHeight = 50;
 
 class HeadingEdit extends Component {
+	constructor() {
+		super( ...arguments );
+		this.splitBlock = this.splitBlock.bind( this );
+	}
+
+	// eslint-disable-next-line no-unused-vars
+	splitBlock( htmlText, start, end ) {
```

Comment:
@daniloercoli , can we perhaps go closer to the interface of the web-side counterpart, wdyt? https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/heading/edit.js#L52.

Reply:
Oh yes, we will probably go toward that direction in the short future. For now i'd like to go with the current implementation, and get the ball rolling.